ci-operator/step-registry/gather/extra/gather-extra-commands.sh

```diff
@@ -105,6 +103,14 @@ queue ${ARTIFACT_DIR}/oc_cmds/subscriptions oc --insecure-skip-tls-verify --requ
 queue ${ARTIFACT_DIR}/clusterserviceversions.json oc --insecure-skip-tls-verify --request-timeout=5s get clusterserviceversions --all-namespaces -o json
 queue ${ARTIFACT_DIR}/oc_cmds/clusterserviceversions oc --insecure-skip-tls-verify --request-timeout=5s get clusterserviceversions --all-namespaces
 queue ${ARTIFACT_DIR}/releaseinfo.json oc --insecure-skip-tls-verify --request-timeout=5s adm release info -o json
+queue ${ARTIFACT_DIR}/clusterroles.json oc --insecure-skip-tls-verify --request-timeout=5s get clusterroles --all-namespaces -o json
+queue ${ARTIFACT_DIR}/oc_cmds/clusterroles_rbac oc --insecure-skip-tls-verify --request-timeout=5s describe clusterroles.rbac
+queue ${ARTIFACT_DIR}/clusterrolebindings.json oc --insecure-skip-tls-verify --request-timeout=5s get clusterrolebindings --all-namespaces -o json
+queue ${ARTIFACT_DIR}/oc_cmds/clusterrolebindings_rbac oc --insecure-skip-tls-verify --request-timeout=5s describe clusterrolebindings.rbac
+queue ${ARTIFACT_DIR}/oc_cmds/clusterrolebindings oc --insecure-skip-tls-verify --request-timeout=5s get clusterrolebindings --all-namespaces
+queue ${ARTIFACT_DIR}/rolebindings.json oc --insecure-skip-tls-verify --request-timeout=5s get rolebindings --all-namespaces -o json
+queue ${ARTIFACT_DIR}/oc_cmds/rolebindings_rbac oc --insecure-skip-tls-verify --request-timeout=5s describe rolebindings.rbac
+queue ${ARTIFACT_DIR}/oc_cmds/rolebindings oc --insecure-skip-tls-verify --request-timeout=5s get rolebindings --all-namespaces
```

Comment:
Same, just the rolebinding names is collected there, already included in previous `get rolebindings --all-namespaces -o json`